File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/SalesOrderDispatchModel.ts
export interface SalesOrderDispatchModel {
    dispatchId: number;
    salesId: number | null;
    dispatchCode: string;
    dispatchDate: Date | null;
    dispatchTime: Date | null;
    shiftDate: Date | null;
    distributorId: number | null;
    carId: number | null;
    driverId: number | null;

    distributorCode: string | null;
    carCode: string | null;
    driverCode: string | null;


    inZone: boolean | null;
    distance: number | null;
    latitude: number | null;
    longitude: number | null;
    feedbackId: number | null;

    notes:string;
}

File: SofiForce.Models/Models/EntityModels/SalesOrderDispatchModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SofiForce.Models.Models.EntityModels
{
    public class SalesOrderDispatchModel
    {

        public long DispatchId { get; set; }
        public long? SalesId { get; set; }
        public string DispatchCode { get; set; }
        public DateTime? DispatchDate { get; set; }
        public DateTime? DispatchTime { get; set; }
        public DateTime? ShiftDate { get; set; }
        public int? DistributorId { get; set; }
        public int? CarId { get; set; }
        public int? DriverId { get; set; }
        public bool? InZone { get; set; }
        public decimal? Distance { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public int? FeedbackId { get; set; }

    }
}
